pkg/agent/agent_windows.go

```diff
@@ -32,3 +40,99 @@ func (i *Initializer) setupExternalConnectivity() error {
 	}
 	return nil
 }
+
+// prepareHostNetworking creates HNS Network for containers.
+func (i *Initializer) prepareHostNetworking() error {
+	return util.PrepareHostNetworking(i.nodeConfig.PodCIDR, i.nodeConfig.NodeIPAddr)
```

Comment:
Not your code, but I feel we should rename util.PrepareHostNetworking to createHNSNetwork or prepareHNSNetwork (util funcs should be generic and not related to Agent logic like container hostNetworking).

Reply:
Done